### DISCUSSION TITLE
Github.com is no accessable.

### DISCUSSION THREAD

**Discussion by akuksin** [community/community#33123] [Other Feature Feedback, Questions, & Ideas]
<img width="1287" alt="image" src="https://user-images.githubusercontent.com/4165556/190316657-a465ddbd-146e-48ec-b4be-1664244f3ff8.png">

**Comment by akuksin** [comment]
user search is not working https://github.com/orgs/<org>/people even the people page is available.

**Comment by melvincwng** [comment]
@akuksin Hey there, I've checked on my end and indeed there seems to be an issue with the GitHub Homepage.

It also seems like other users are experiencing the same issue as well, as seen from the other discussion threads.

Hopefully the support engineers at GitHub are able to resolve this issue asap.

**Reply by haquenafeem** [reply]
for now you can also try by github.com/<yourusername>

**Comment by DSoftwareArtist** [comment]
Other pages works except the homepage. Have a try!
